packages/webvh/src/anoncreds/services/WebVhAnonCredsRegistry.ts

```diff
@@ -460,4 +607,116 @@ export class WebVhAnonCredsRegistry implements AnonCredsRegistry {
       return false
     }
   }
+
+  public async createProof(
+    agentContext: AgentContext,
+    unsecuredDocument: UnsecuredDocument,
+    verificationMethod: string
+  ) {
+    const cryptosuite = new EddsaJcs2022Cryptosuite(agentContext)
+    try {
+      const creationResult = await cryptosuite.createProof(unsecuredDocument, {
+        type: 'DataIntegrityProof',
+        cryptosuite: 'eddsa-jcs-2022',
+        verificationMethod,
+        proofPurpose: 'assertionMethod',
+      })
+      return creationResult
+    } catch (error) {
+      agentContext.config.logger.error('Error during proof creation of did:webvh resource', {
+        error: error instanceof Error ? error.message : String(error),
+        stack: error instanceof Error ? error.stack : undefined,
+      })
+    }
+  }
+
+  private async buildSignedResource(
+    agentContext: AgentContext,
+    {
+      content,
+      id,
+      metadata,
+      issuerId,
+      verificationMethod,
+    }: {
+      content:
+        | AnonCredsSchema
+        | AnonCredsCredentialDefinition
+        | AnonCredsRevocationRegistryDefinition
+        | AnonCredsRevocationStatusListWithoutTimestamp
+      id: string
+      metadata: Record<string, unknown>
+      issuerId: string
+      verificationMethod?: string
+    }
+  ) {
+    const verificationMethodId = await this.getVerificationMethodId(agentContext, issuerId, verificationMethod)
+
+    // Prepare the generic resource payload to be signed
+    const resourcePayload = {
+      '@context': ['https://opsecid.github.io/attested-resource/v1', 'https://w3id.org/security/data-integrity/v2'],
```

Comment:
```suggestion
      '@context': ['https://identity.foundation/did-attested-resources/context/v0.1', 'https://w3id.org/security/data-integrity/v2'],
```
The spec is now published and the context is available at this endpoint